charts/azuremonitor-containers/values.yaml

```diff
@@ -10,9 +10,13 @@ Azure:
     Cloud: <your_cluster_cloud>
     Region: <your_cluster_region>
     ResourceId: <your_cluster_id>
+    Distribution: ""  # Will be populated with "openshift" for OpenShift clusters
   Extension:
     Name: ""
     ResourceId: ""
```

Comment:
This PR also missing defining security context constraint resource and binding to the ama-logs daemonset and deployment pods.